docs/9-membership.md

```diff
@@ -149,156 +157,15 @@ For transparency purposes, we are disclosing each Guild member's team, the name/
  | TXRX | [Alex Vlasov](https://github.com/ericsson49) | 
  | TXRX | [Anton Nashatyrev](https://github.com/Nashatyrev) | 
  | TXRX | [Mikhail Kalinin](https://github.com/mkalinin) | 
+ | ConsenSys DDS | [Chenyi Zhang](https://github.com/czhang-fm) | 
+ | ConsenSys DDS | [Roberto Saltini](https://github.com/saltiniroberto) | 
```

Comment:
If this list is alphabetically, these two folks should be at the top.

Also, `ConsenSys` has [recently rebranded](https://github.com/Consensys) to `Consensys` (lower case "s").

Reply:
Good catch! Both updated.